cicd/gitops/gitops-release-notes.adoc

```diff
@@ -23,14 +23,20 @@ include::modules/go-compatibility-and-support-matrix.adoc[leveloffset=+1]
 include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
 
 // Modules included, most to least recent
+include::modules/gitops-release-notes-1-7-1.adoc[leveloffset=+1]
+
 include::modules/gitops-release-notes-1-7-0.adoc[leveloffset=+1]
 
+include::modules/gitops-release-notes-1-6-4.adoc[leveloffset=+1]
+
 include::modules/gitops-release-notes-1-6-2.adoc[leveloffset=+1]
 
 include::modules/gitops-release-notes-1-6-1.adoc[leveloffset=+1]
 
 include::modules/gitops-release-notes-1-6-0.adoc[leveloffset=+1]
 
+include::modules/gitops-release-notes-1-5-9.adoc[leveloffset=+1]
```

Comment:
**Observation:** [GITOPS-1756](https://issues.redhat.com/browse/GITOPS-1756) is already documented in 4.8 OCP as Fixed Issues. 

**Suggestion:** 

- So no need of repeating it in RN 1.4.2 and 1.3.3. 
- Also, add the RN module 1.2.2 in OCP version 4.9 and later.

Reply:
Incorporated this suggestion @Srivaralakshmi. Please verify and confirm. Thanks